### PR TITLE
Set Up To-Do List: Add and Remove

### PR DESCRIPTION
**Pull request by angeluray**
## Hi good reviewer,

Patricio greeting

## Basic Check✔

- Created a new branch according to the Github Flow rules.
- Added a CSS file that includes styles for To-Do list new features.
- Added a JS file that includes the logic process and structure.
- Set up linters for HTML/CSS and JS making sure they work correctly.
- Added a .gitignore file to make sure any irrelevant files are ignored by git.
- Built using the Webpack.

## Project Requirements Basic checks ✔

- Remove all hardcoded items from the tasks array. - **Done.**
- Create a new JavaScript file for the new functionality. **Done.**
- Implement a function for adding a new task (add a new element to the array). - **Done.**
- Implement a function for deleting a task (remove an element from the array). - **Done.**
- Implement a function for editing task descriptions. - **Done.**
- By default new tasks should have the property completed set to false and the property index set to the value of the new array length (i.e. if you're adding a 5th task to the list, the index of that task should equal to 5). - **Done.**
- Deleting a task should update all remaining items' indexes, so they represent the current list order and are unique(i.e. if you're deleting the first task index 1 from the list, the index of the next task(2) should set to 1).. - **Done.**
- All changes to the To Do List should be saved in local storage. - **Done.**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -22,58 +28,17 @@ refreshIcon.classList.add('mySendIcon');
...
-  document.getElementsByClassName('list')[0].innerHTML = input.map((items, index) => `
-  <li id="${index}"><div><input type="checkbox" class="${items.myBool}">${items.listInfo}</div><img src='https://img.icons8.com/windows/32/000000/menu-2.png' class='myOptionIcon'></li>`).join('');
-};
-
 const addNewList = () => {
-  counter += 1;
   const toDo = document.getElementById('toDo').value;
-  const eachItemList = new NewItemList(toDo, false, counter);
+  const eachItemList = new NewItemList(toDo, false, (eachList.length - 1 + 1));
   // Push into the array
```
Comment (mahbubul14) [inline src/index.js]: - [ ] By default new tasks should have the property index set to the value of the new array length (i.e. if you're adding a 5th task to the list, the index of that task should equal to 5). Kindly fix the issue.

**Example:**  `const eachItemList = new NewItemList(toDo, false, (eachList.length + 1));`

**Review by mahbubul14 — CHANGES_REQUESTED** [review]
Hi @angeluray ,

Good job so far! :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

# Required Changes ♻️

- [ ] Kindly check the inline comments under the review. :arrow_down: 

###  To Highlights
:heavy_check_mark: No linter error.
:heavy_check_mark: Good README file.
:heavy_check_mark: Descriptive PR title & description.

### Optional suggestions

* N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also reach me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/index.js (hunk 1 above)
Comment (angeluray) [reply src/index.js]: Hi @mahbubul14, thank you for your feedback. I already changed that.

**Review by whiz25 — APPROVED** [review]
Hi @angeluray, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

### Martin Afani
#### Slack: whiz25

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**